Sources/SwiftSyntax/Raw/RawSyntaxArena.swift

```diff
@@ -30,27 +30,27 @@
 /// significantly speeds up parsing.
 ///
 /// As a consequence, syntax nodes cannot be freed individually but the memory
-/// will get freed once the owning ``SyntaxArena`` gets freed. Thus, it needs to
-/// be manually ensured that the ``SyntaxArena`` is not deallocated while any
+/// will get freed once the owning ``RawSyntaxArena`` gets freed. Thus, it needs to
+/// be manually ensured that the ``RawSyntaxArena`` is not deallocated while any
 /// of its nodes are being accessed. The `SyntaxData` type ensures this as
 /// follows:
-/// - The root node has a strong reference to its ``SyntaxArena``
+/// - The root node has a strong reference to its ``RawSyntaxArena``
 /// - Each node retains its parent `SyntaxData`, thus keeping it alive.
-/// - If any node is allocated within a different ``SyntaxArena``,  that arena
+/// - If any node is allocated within a different ``RawSyntaxArena``,  that arena
 ///   is added to the root's `childRefs` property and thus kept a live as long
 ///   as the parent tree is alive.
 ///
-/// As an added benefit of the ``SyntaxArena``, `RawSyntax` nodes don’t need to
+/// As an added benefit of the ``RawSyntaxArena``, `RawSyntax` nodes don’t need to
 /// be reference-counted, further improving the performance of ``SwiftSyntax``
 /// when worked with at that level.
-public class SyntaxArena {
+public class RawSyntaxArena {
```

Comment:
It's unfortunate that `SyntaxArena` was non-SPI `public`. I'm going to add deprecated `typealias SyntaxArena = RawSyntaxArena`